### PR TITLE
replace ant by chakra

### PR DESCRIPTION
**Pull request by internetti**
Jira ticket: JIRA-CORE24-#

## What have you implemented?

replaced ant by chakra ui

## How to test it?

run the app frontend, should have a basic themed layout

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,14 +12,17 @@
...
+    "@emotion/react": "^11.11.4",
+    "@emotion/styled": "^11.11.5",
     "axios": "^1.6.8",
     "connect-session-knex": "^4.0.0",
     "cors": "^2.8.5",
     "dotenv": "^16.4.5",
     "express": "^4.19.2",
     "express-rate-limit": "^7.2.0",
     "express-session": "^1.18.0",
+    "framer-motion": "^11.1.7",
```
Comment (neb42) [inline package.json:25]: Does chakra need framer motion? Ideally we'd disable animations (for performance) and remove this (for the bundle size)

File: libs/theme/project.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "build": {
+      "executor": "@nx/js:tsc",
+      "outputs": ["{options.outputPath}"],
+      "options": {
+        "outputPath": "dist/libs/theme",
+        "main": "libs/theme/src/index.ts",
+        "tsConfig": "libs/theme/tsconfig.lib.json",
+        "assets": ["libs/theme/*.md"]
+      }
+    }
```
Comment (neb42) [inline libs/theme/project.json:16]: I think we can remove the build target

**Review by neb42 — APPROVED** [review]

File: libs/theme/README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# theme
+
+This library was generated with [Nx](https://nx.dev).
```
Comment (dfigueroa87) [inline libs/theme/README.md:3]: I think would be good to start adding some details about the libs in here. What are they for, how to use them or any relevant information

**Review by dfigueroa87 — APPROVED** [review]
Looks good, just one minor general comment

File: package.json (hunk 1 above)
Comment (internetti) [reply package.json:25]: build fails without it

Comment (internetti) [reply package.json:25]: it might be able to turn of, but first google looks like we'll still have to install it

File: libs/theme/README.md (hunk 3 above)
Comment (neb42) [reply libs/theme/README.md:3]: I think this goes for most libs we've created. I know I've been bad for this

**Pull request merged**